UtopiaCity/Models/Common/ExtendedEntity.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace UtopiaCity.Models.Common
+{
+    public class ExtendedEntity: BaseEntity
```

Comment:
If we would want to use for all models stored in db we can make name which reflects its project allocation. E.g. UtopiaBaseEntity, UEntity, etc. 
Just a suggestion though